proto/mpp.proto

```diff
@@ -19,6 +19,12 @@ message TaskMeta {
 	string address = 4; // target address of this task.
 }
 
+message IsAliveRequest {
+}
+
+message IsAliveResponse {
+}
```

Comment:
Should TiFlash send the response if it is not down, but it can not handle mpp request(for example, the server is in terminating state)?

Reply:
It's a little complicated. I suggest leave it like this.